Packages/com.unity.inputsystem/InputSystem/Editor/PropertyDrawers/InputActionDrawerBase.cs

```diff
@@ -144,6 +209,12 @@ private void OnGUI()
 
             private PropertiesViewBase m_PropertyView;
         }
+
+        private class InputActionDrawerViewData
```

Comment:
Nitpick: struct seems to do just as well.

Reply:
If I change it to struct, I'd have to change GetOrCreateViewData to have a ref argument because the instance it returns is modified in various places. I can do that, but I think it's cleaner as is. Thoughts?

Reply:
Totally up to you. In my head, paying 36 bytes for an object header plus 8 bytes for a pointer on a payload of 16 bytes just brings up an immediate "hmmm....". But in the context here, even classifying it as a "nitpick" is inflating this *massively*. Matter of fact, I most likely wasted two persons' mental cycles on absolutely nothing here :D

Reply:
Haha, it's good to be critical. Off the hot path, I think maintainability and readability are more important than squeezing every byte, so if you can still sleep at night with this in the code, I'll leave it as is.